### DISCUSSION TITLE
Scan covers, and upload covers to Open Library

### DISCUSSION THREAD

**Discussion by lee-b** [mateusz-bak/openreads#740] [Ideas]
It would be really nice to be able to scan covers for the books that you already have, a la [Genius Scan](https://play.google.com/store/apps/details?id=com.thegrizzlylabs.geniusscan.free&hl=en_US). Even better if we could then upload the covers to Open Library, helping everyone.

**Comment by lucastucious** [comment]
Should be done with the #85
